app/src/main/java/com/framgia/soundclound/util/Constant.java

```diff
@@ -39,5 +40,10 @@
     public static final String EXTRA_TRACK = "com.frangia.soundcloud.EXTRA_TRACK";
     public static final int INDEX_DEFAULTE = 1;
     public static final String FILE_EXTENTION = ".mp3";
-}
 
+    public static int FOREGROUND_SERVICE = 101;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName